src/components/hero.mdx

```diff
@@ -0,0 +1,33 @@
+<div className="cmp-hero">
+  <h1 className="cmp-hero__heading">
+    <a className="cmp-hero__logo" href="https://seesparkbox.com">
+      <img src="/images/logo.svg" alt="Sparkbox" />
+    </a>
+    <span className="cmp-hero__title">Design Systems</span>
+    <span className="cmp-hero__survey">Survey</span>
+    <span className="cmp-hero__year">2019 Edition</span>
+  </h1>
+  <div className="cmp-hero__flood" role="image"></div>
```

Comment:
The validator is not liking these `role="image"`: 
<img width="646" alt="Screen Shot 2019-07-17 at 10 45 05 AM" src="https://user-images.githubusercontent.com/23404383/61385203-f5395900-a87f-11e9-8816-5c84b7dc306c.png">